dist/js/script.js

```diff
@@ -3,3 +3,49 @@
  * Date: 12 - 06 -2021
  * Copyright (c) 2021 Calvince
  */
+
+(function () {
+  // Utility functions
+
+  // /**
+  //  * @param {string} idValue - an element ID
+  //  * @returns {Object} DOM object associated with id
+  //  * */
+  // function id(idName) {
+  //   return document.getElementById(idName);
+  // }
+
+  /**
+   * @param {string} className - a classs name of the html elements
+   * @returns {Object} Dom object(s) of the associated html elements
+   */
+  function classItem(className) {
+    return document.querySelector(className);
+  }
+
+  // Variables
+  let isOpen = false;
+  const menu = classItem('.mobile-menu-button');
+  const menuItem = classItem('.mobile-menu-button-item');
+  const nav = classItem('.desktop-menu');
```

Comment:
- [ ] **Please make sure that the menu matches the original design from Figma.**

The menu in original design spans over the entire height of the viewport. Please modify your menu to match the original design 

Cloned Design|Orginal Design
:------------------:|:-------------------:
![image](https://user-images.githubusercontent.com/59168430/123180401-cf278500-d4a8-11eb-8328-2327a5792900.png)|![image](https://user-images.githubusercontent.com/59168430/123180453-ed8d8080-d4a8-11eb-962a-8b4c18219a70.png)